path_role.go

```diff
@@ -426,8 +436,31 @@ func (b *jwtAuthBackend) pathRoleCreateUpdate(ctx context.Context, req *logical.
 		role.VerboseOIDCLogging = verboseOIDCLoggingRaw.(bool)
 	}
 
+	boundClaimsType := data.Get("bound_claims_type").(string)
+	if boundClaimsType != boundClaimsTypeString && boundClaimsType != boundClaimsTypeGlob {
```

Comment:
Suggested refactoring:

```
switch boundClaimsType {
    case boundClaimsTypeString, boundClaimsTypeGlob:
        role.BoundClaimsType = boundClaimsType
    default:
        return logical.ErrorResponse("invalid 'bound_claims_type': %s", boundClaimsType), nil
}
```

Reply:
Implemented the change in the new commit (to simply reviewing it)